spec/forms/providers/application_merits_task/matter_opposed_form_spec.rb

```diff
@@ -0,0 +1,54 @@
+require "rails_helper"
+
+RSpec.describe Providers::ApplicationMeritsTask::MatterOpposedForm do
+  let(:legal_aid_application) { create(:legal_aid_application) }
+  let(:params) { { model: build(:matter_opposition, legal_aid_application:), matter_opposed: } }
+
+  describe "#validate" do
+    subject(:form) { described_class.new(params) }
+
+    context "when matter_opposed is blank" do
+      let(:matter_opposed) { nil }
+
+      it "is invalid" do
+        expect(form).not_to be_valid
+        expect(form.errors).to be_added(:matter_opposed, :inclusion, value: nil)
+      end
+    end
+
+    context "when matter_opposed is not blank" do
+      let(:matter_opposed) { "true" }
+
+      it "is valid" do
+        expect(form).to be_valid
+      end
+    end
```

Comment:
Do we generally test both (true and false) valid options?

Reply:
I left it in as it was changing the current behaviour

Reply:
Is @naseberry suggesting you add another test for `let(:matter_opposed) { "false" }`?